sklearn/decomposition/_truncated_svd.py

```diff
@@ -85,17 +85,18 @@ class TruncatedSVD(TransformerMixin, BaseEstimator):
     Examples
     --------
     >>> from sklearn.decomposition import TruncatedSVD
+    >>> import scipy.sparse as sp
```

Comment:
We use `import scipy.sparse as sp` in this very file.

I don't think this is confusing as long as the example is self contained

Reply:
I think readers will find this confusing. I would prefer

```py
import scipy.sparse as sparse

# or

from scipy.sparse import random
```

Reply:
I'll change it, but since I'm also stubborn ;) :

```
$ git grep "import scipy.sparse as sp" | wc -l 
67
```

We also use it in the UG already https://scikit-learn.org/stable/modules/impute.html#univariate-feature-imputation

Reply:
used `from scipy.sparse import random as sparse_random`